src/plotting/task_plot_scenario_comparisons.py

```diff
@@ -347,7 +356,7 @@ def task_plot_scenario_comparison(
         "share_ever_rapid_test",
         "share_rapid_test_in_last_week",
     ]
-    if empirical is None:
+    if empirical is True:
```

Comment:
Maybe ``show_empirical`` if I understand the purpose correctly?

```suggestion
    if empirical:
```

Reply:
The inclusion of empirical time series is really not well implemented at the moment with code doing similar things in many places. I'll redo this in a future PR.